Common/UnitDefinitions/WarpingMomentOfInertia.json

```diff
@@ -87,8 +87,8 @@
       "BaseUnits": {
         "L": "Inch"
       },
-      "FromUnitToBaseFunc": "{x} * Math.Pow(2.54e-2, 6)",
-      "FromBaseToUnitFunc": "{x} / Math.Pow(2.54e-2, 6)",
+      "FromUnitToBaseFunc": "{x} * 0.000000000268535866540096",
+      "FromBaseToUnitFunc": "{x} / 0.000000000268535866540096",
```

Comment:
In a way, I kind of prefer the Math.Pow formula, since it is easier to recognize the constant 2.54 and even repeating it 6 times `2.54e-2 * 2.54e-2 * ...` could be one option if Math.Pow throws a curveball.
I have no strong opinion here, just an observation.

Reply:
I can work with `Math.Pow` as well, but if we (or whoever) wanted to created a generator for another language, the uses of `Math` means more hoops to deal with..